ts/docs.ts

```diff
@@ -48,49 +86,116 @@ async function docs(): Promise<void> {
+          log.warn(
+            `docs.ts#57: cant decode '${mdFile}' as 'Article'; skipping HTML generation`
+          )
+          return
+        }
 
-          // Don't output an HTML file if the Markdown document cannot be decoded into an article
-          if (!isArticle(article)) {
-            return
-          }
+        log.debug(`schema2Article(${jsonFile}) => ${schemaMdFile}`)
+        if (!mdFileExists) {
+          stats.htmlFromYamls++
+          log.info(
+            `(${stats.htmlFromYamls})${maybeMdFile} doc not found; stubbing html from schema2Article(${mdFile})`
+          )
+        }
 
-          const processed = await encodaProcess(
+        // dom fragment to let readers know this file was generated automatically
+        const subheadingContent = !mdFileExists
+          ? [
+              {
+                type: 'Heading',
+                depth: 6,
+                content: [
+                  `note: this html file was autogenerated by 'ts/docs.ts::schema2Article()' from '${schemaMdFile}'; create documentation for it at '${maybeMdFile}'`
+                ],
+                id: 'notice'
+              }
+            ]
+          : []
+
+        // can we find any examples to include?
+        // note: path-matching in getExamples() that looks for .yaml examples
+        // is not too reliable b/c I am not sure what the naming pattern is
+        const yamlExamples = await getExamples(title)
+        let exampleContent = yamlExamples.map((example: any) => {
+          return {
+            type: 'CodeBlock',
+            programmingLanguage: 'yaml',
+            text: example
+          }
+        })
+        if (exampleContent.length > 0) {
+          exampleContent = [
             {
-              ...article,
-              title,
+              type: 'Heading',
+              depth: 2,
+              content: ['Examples'],
+              id: 'notice'
+            },
+            {
+              type: 'Heading',
+              depth: 5,
               content: [
-                {
-                  type: 'Heading',
-                  depth: 2,
-                  content: [
-                    'Status: ',
-                    schema.status,
-                    ', ',
-                    'Role: ',
-                    schema.role
-                  ]
-                },
-                ...(article.content || [])
-              ]
+                `(auto-included from "examples/${title}/*.${title}.yaml")`
+              ],
+              id: 'notice'
             },
-            path.dirname(mdFile)
-          )
-
-          const htmlFile = path.join('public', `${title}.html`)
-          await encoda.write(processed, htmlFile, {
-            isBundle: false, // Set isBundle to true to work locally with NPM linked Thema style changes
-            theme: 'stencila'
-          })
+            ...exampleContent
+          ]
         }
+        if (yamlExamples.length > 0) stats.examplesInserted++
```

Comment:
To make sure I understand the intent of this section, should the `stats.examplesInserted` be incremented by `1`, or the value of `yamlExamples.length`? If the latter, it can also be merged into the above `yamlExamples.map(` statement.